src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx

```diff
@@ -870,93 +890,90 @@ By default, the attributes are sorted by frequency with the least frequently rep
 
 ### How does topology correlation work? [#topology-explained]
 
 <img
   title="topology-4.png"
-  alt="New Relic topology explained 4"
+  alt="A screenshot of New Relic topology explained"
   src={topology4}
 />
 
 <figcaption>
   In this service map, the hosts and apps are the vertices, and the lines showing their relationships are the edges.
 </figcaption>
 
-Topology correlation relies on two main concepts:
+To set up your topology in addition to the [entities and relationships](/docs/new-relic-solutions/new-relic-one/core-concepts/what-entity-new-relic/) collected by [New Relic agents](/docs/new-relic-solutions/new-relic-one/install-configure/compatibility-requirements-new-relic-agents-products/), use our [NerdGraph API](#create-topology-graph). 
+
+Customized topology correlation relies on two main concepts:
 
-* **Vertex**: A vertex represents a monitored entity: it's the source from which your incident events are coming from, or describing a problematic symptom about. A vertex has attributes (key/value pairs) configured for it, like entity GUIDs or other IDs, which allow it be associated with incoming incident events.
-* **Edges**: An edge is a connection between two vertices. Edges describe the relationship between vertices.
+* **Vertex:** A vertex represents a monitored entity. It's the source from which your incident events are coming from, or describing a problematic symptom about. A vertex has attributes (key/value pairs) configured for it, like entity GUIDs or other IDs, which allow it be associated with incoming incident events.
+* **Edges:** An edge is a connection between two vertices. Edges describe the relationship between vertices.
 
-To set up your topology, you'll use [our NerdGraph API](#create-topology-graph). But first, it may help to understand how topology is used to correlate incidents:
+It may help to understand how topology is used to correlate incidents:
 
-1. First, New Relic gathers all relevant incidents. This includes incidents where [decision-logic steps 1 and 2](#build-decision) are true, and that are also within the defined time window in advanced settings.
+1. First, New Relic gathers all relevant incidents. This includes incidents where [decision logic steps 1 and 2](#build-decision) are true and that are also within the defined time window in advanced settings.
+
+  In this example, all of the incidents in the dotted-line selection have met these requirements: they've gone through the [decision logic in steps 1 and 2](#build-decision), and all contextual comparisons made in [decision logic step 2](#build-decision) are true.
 
```

Comment:
Not sure where the anchors in this step are supposed to go, but they don't work in Gatsby preview. Are they really needed?

Reply:
@barbnewrelic, thanks for catching this oversight! I think these were all supposed to point to the "custom" anchor, but I will confirm.